db/migrate/20200805015237_user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ActiveRecord::Migration[6.0]
```

Comment:
Because there's already a class named `User` in model this migration runs into an error `TypeError: superclass mismatch for class User` and unfortuantely the app won't run without successful migration runs.